swiftpkg/bzlmod/swift_deps.bzl

```diff
@@ -91,17 +225,17 @@ _configure_package_tag = tag_class(
             mandatory = True,
         ),
         "recursive_init_submodules": attr.bool(
-            default = True,
+            default = False,
```

Comment:
Why the change?

Reply:
This was an accident. Must have happened during a rebase.